media/css/firefox/whatsnew/whatsnew-117-eu-reader-view.scss

```diff
@@ -0,0 +1,326 @@
+    display: inline-block;
+    height: 2ex;
+    margin-bottom: -0.25ex;
+    width: 2ex;
+}
+
+// ---------------------------------------------------------------------------*/
+// All the bells and whistles
+
+@media #{$mq-md} and (prefers-reduced-motion: no-preference) {
+    .wnp-grid {
+        display: grid;
+        grid-gap: $spacing-md;
+        grid-template-columns: 1fr 3fr 1fr;
+        grid-template-rows: 90px 1fr 90px;
+    }
+
+    .wnp-sign-off {
+        grid-column: 1 / span 3;
+        grid-row: 3;
+    }
+
+    .wnp-grid-body {
+        grid-column: 2;
+        grid-row: 2;
+        max-width: none;
+        padding: $spacing-2xl $spacing-lg;
+    }
+
+    .wnp-grid-head {
+        display: block;
+        grid-column: 2;
+        grid-row: 1;
+    }
+
+    .wnp-grid-col1 {
+        display: grid;
+        grid-column: 1;
+        grid-gap: $spacing-md;
+        grid-row: 1 / span 2;
+        grid-template-rows: 1fr 2fr 1.5fr;
+    }
+
+    .wnp-grid-col2 {
+        display: grid;
+        grid-column: 3;
+        grid-gap: $spacing-md;
+        grid-row: 1 / span 2;
+        grid-template-rows: 2fr 1fr;
+    }
+
+    .wnp-content-main {
+        animation: mainbg 500ms 3250ms ease-in forwards;
+        background-color: var(--main-bg-color);
+    }
+
+    .wnp-main-title {
+        margin-top: -$layout-lg;
+
+        &::before {
+            animation: fade-in 300ms 3200ms ease-in forwards, pop 400ms 3200ms ease-in forwards;
+            transform: scale(0.9);
+            opacity: 0;
+        }
+    }
+
+    .eyesore {
+        background-color: var(--eyesore-bg-color);
+        box-sizing: border-box;
+        color: var(--eyesore-text-color);
+        padding: $spacing-md $spacing-lg;
+        position: relative;
+        text-align: start;
+    }
+
+    .eyesore-one {
+        animation: eyesore-one 1000ms 2500ms ease-in forwards, fade-out 500ms 3500ms ease-out forwards;
+        min-height: 90px;
+
+        span {
+            left: $spacing-lg;
+            max-width: 60%;
+            position: absolute;
+            top: $spacing-md;
+
+            &::before {
+                @include box-decoration-break(clone);
+                background: var(--eyesore-text-color);
+                color: transparent;
+                content: "The 91 best clickbait listicles of the week so far. Number 36 will shock you!";
+                display: inline;
+                font-size: 12px;
+            }
+        }
+
+        &::after {
+            background: var(--eyesore-text-color);
+            border-radius: $border-radius-sm;
+            content: "";
+            height: 2em;
+            position: absolute;
+            right: $spacing-lg;
+            width: 20%;
+        }
+    }
+
+    .eyesore-two,
+    .eyesore-three,
+    .eyesore-five {
+        align-items: center;
+        display: flex;
+        justify-content: center;
+
+        &::before {
+            @include text-title-xs;
+            content: "Ad";
```

Comment:
Just a thought: should we localize this label for DE and FR? According to Google translate a German equivalent would be "Anzeige" and French would be "Annonce." It's essentially decorative and much of the audience likely understands "Ad" in this context anyway. If we did change it, we'd likely want to run it by the writers first and it's already been approved so not worth starting a whole debate, but I just wanted to mention it.